### PR TITLE
Only copy apps from erts path if the app is part of erts

### PR DESCRIPTION
**Pull request by mobileoverlord**
I am open to structuring this differently :)

### CODE REVIEW

File: lib/mix/lib/releases/assembler.ex [hunk 1]
```diff
@@ -160,6 +166,7 @@ defmodule Mix.Releases.Assembler do
   end
 
   defp copy_app(app_dir, target_dir, false, include_src?) do
+    IO.inspect app_dir
```
Comment (bitwalker) [inline lib/mix/lib/releases/assembler.ex]: You forgot something :P

**Comment by bitwalker** [comment]
I'm +1, just need to remove that inspect :)

File: lib/mix/lib/releases/assembler.ex (hunk 1 above)
Comment (mobileoverlord) [reply lib/mix/lib/releases/assembler.ex]: 😇

**Comment by mobileoverlord** [comment]
Rebased, Whoops :)

**Comment by mobileoverlord** [comment]
blaergh, stupid space

**Comment by mobileoverlord** [comment]
Okay, _this_ time!

**Pull request merged**